### PR TITLE
spinning until service discovery instances are there

### PR DESCRIPTION
**Pull request by lamdor**

### CODE REVIEW

**Comment by trentio** [comment]
:+1:

File: lib/rspec/sidecar/services.rb [hunk 1]
```diff
@@ -15,9 +15,14 @@ def service(name, type,
...
-      instance_json = zk.get(instances_path + "/" + instances[0])[0]
-      JSON.parse(instance_json)
+      loop do
+        instances = zk.children(instances_path)
+        unless instances.empty?
+          instance_json = zk.get(instances_path + "/" + instances[0])[0]
+          return JSON.parse(instance_json)
+        end
+        sleep 1
+      end
```
Comment (DustinChaloupka) [inline lib/rspec/sidecar/services.rb:25]: What happens if an image doesn't work right and no service ever registers itself with zookeeper?  Won't this just spin forever and if it's in a jenkins build someone will just have to notice?

Comment (lamdor) [inline lib/rspec/sidecar/services.rb:25]: That's a good catch. I'll fix.

Comment (lamdor) [inline lib/rspec/sidecar/services.rb:25]: @DustinChaloupka fixed in https://github.com/Banno/rspec-sidecar/pull/4

**Pull request merged**